tendermint/src/lite/types.rs

```diff
@@ -101,13 +101,13 @@ pub trait Commit {
     fn votes_len(&self) -> usize;
 }
 
-/// TrustThreshold defines what fraction of the total voting power of a known
+/// [`TrustThreshold`] defines what fraction of the total voting power of a known
 /// and trusted validator set is sufficient for a commit to be
 /// accepted going forward.
 /// The default implementation returns true, iff at least a third of the trusted
 /// voting power signed (in other words at least one honest validator signed).
 /// Some clients might require more than +1/3 and can implement their own
-/// TrustLevel which can be passed into all relevant methods.
+/// [`TrustLevel`] which can be passed into all relevant methods.
```

Comment:
Looks like we forgot to rename `TrustLevel` to `TrustThreshold` here. Running `cargo +nightly doc --open --no-deps` will error now with something like 
```
`[TrustLevel]` cannot be resolved, ignoring it.
```
because of  a `#[deny(warnings)]`. 

Does this mean building for https://docs.rs/ won't work now?

Reply:
I'm not sure if docs.rs will break if you have intra-doc resolution failures and deny warnings but possibly.

At any rate, it's good to capture and correct these. Running `cargo test --doc` on nightly will catch them.